pipeline-scripts/release.groovy

```diff
@@ -452,6 +456,39 @@ def void sendReleaseCompleteMessage(Map release, int advisoryNumber, String advi
     }
 }
 
+def createAdvisoriesFor(ocpVersion) {
+    build(
+        job: "/build/advisories",
+        propagate: false,
+        parameters: [
+            string(name: "VERSION", value: ocpVersion),
+            string(name: "DATE",    value: determineNextReleaseDate(ocpVersion)),
+            string(name: "IMPETUS", value: "standard"),
+        ]
+    )
+}
+
+def determineNextReleaseDate(ocpVersion) {
+    def dayOfWeek
+    switch(ocpVersion) {
+        case "4.5":
+            dayOfWeek = DayOfWeek.MONDAY
+            break
+        case "4.4":
+            dayOfWeek = DayOfWeek.TUESDAY
+            break
+        default:
+            dayOfWeek = DayOfWeek.WEDNESDAY
+            break
+    }
+    return (
+        LocalDate.now()
+        .with(TemporalAdjusters.next(dayOfWeek))
+        .with(TemporalAdjusters.next(dayOfWeek))            // 2 weeks from now
```

Comment:
I am getting all kinds of questions about java's understanding of "next Wednesday" depending on what day it is today.